google/cloud/bigtable/examples/opencensus/bigtable_opencensus.cc

```diff
@@ -0,0 +1,166 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! [all code]
+
+//! [bigtable includes]
+#include "google/cloud/bigtable/table.h"
+#include "google/cloud/bigtable/table_admin.h"
+//! [bigtable includes]
+
+#ifndef OPENCENSUS_DISABLED
```

Comment:
Where is this macro defined?  It looks like it is not and this code only compiles if OpenCensus is available, please remove.

Reply:
removed.